MH4U Dex/Base.lproj/Main.storyboard

```diff
@@ -156,6 +156,15 @@
                                     <segue destination="hfi-eK-gsg" kind="show" id="dX6-Hf-8Bu"/>
                                 </connections>
                             </button>
+                            <button opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Cxa-Tg-f1x">
+                                <rect key="frame" x="35" y="377" width="136" height="30"/>
+                                <state key="normal" title="RESET CORE DATA">
```

Comment:
Just a general note: make sure when you're setting titles of things in storyboards that you're also making sure you're using appropriate methods to localize - otherwise things may not get localized properly.